sdk/android/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/table/sync/localstore/MobileServiceLocalStore.java

```diff
@@ -71,18 +71,20 @@
      *
      * @param tableName the local table name
      * @param item      the item to be inserted
+     * @param fromServer
```

Comment:
For completeness can you add the doc for the 'fromServer' parameter?  In managed we have:
"-true- if the call is made based on data coming from the server e.g. in a pull operation; -false- if the call is made by the client, such as insert or update calls on an IMobileServiceSyncTable.
"